tests/BladeTest.php

```diff
@@ -142,6 +142,16 @@ public function the_hasanyrole_directive_will_evaluate_true_when_the_logged_in_u
     }
 
     /** @test */
+    public function the_hasanyrole_directive_will_evaluate_true_when_the_logged_in_user_does_have_some_of_the_required_roles_use_pipe_for_the_given_guard()
+    {
+        $guard = 'admin';
+
+        auth('admin')->setUser($this->getSuperAdmin());
+
+        $this->assertEquals('does have some of the roles', $this->renderView('guardHasAnyRolePipe', compact('guard')));
```

Comment:
Could you also add a test that checks if `does not have any of the given roles` is returned when a user has none of the requested roles?